src/main/resources/application-local.yml

```diff
@@ -66,6 +67,6 @@ user-allocations:
         wales: JIMSNOWLDAP
         nat: JIMSNOWLDAP
     reallocated-assessments:
-      enabled: true
+      enabled: false
     female-assessments:
-      enabled: true
+      enabled: false
```

Comment:
This appears to be 'true' in the helm configs for dev/prod, so maybe should leave as true?